### ISSUE TITLE
The expand indicator changes after scrollling

### ISSUE THREAD

**Issue by nikhiljadhav159** [thoughtbot/expandable-recycler-view#135]
Once group is expanded the after scrolling the expanded group is expanded but the indicator changes.

**Comment by athira09** [comment]
facing same issue. have you got a solution?

**Comment by nikhiljadhav159** [comment]
Yes, I have managed it using the Boolean variable as isExpanded this is custom variable.
